flake8_quotes/__init__.py

```diff
@@ -44,19 +44,27 @@ def __init__(self, tree, filename='(none)', builtins=None):
     def add_options(cls, parser):
         parser.add_option('--quotes', default='\'', action='store',
                           help='Quote to expect in all files (default: \')')
-        parser.add_option('--multiline-quotes', default='\'', action='store',
-                          help='Multiline quote to expect in all files (default: \')')
+        parser.add_option('--multiline-quotes', default='', action='store',
+                          help='Multiline quote to expect in all files (default: any)')
         parser.config_options.extend(['quotes', 'multiline_quotes'])
 
     @classmethod
     def parse_options(cls, options):
+        multiline_ignored = True
+
         if hasattr(options, 'multiline_quotes'):
             multiline_quotes = options.multiline_quotes
+
+            if len(multiline_quotes) == 0:
+                multiline_quotes = options.quotes
+            else:
+                multiline_ignored = False
```

Comment:
Ah, I think the logic was a bit fucky there. We shouldn't care about the user's option if they want us to ignore it. Let's try the following:

```
# Define a new constant named `MULTILINE_QUOTE_STRS
# This should be after MULTILINE_QUOTES
MULTILINE_QUOTE_STRS = ('"""', '\'\'\'')

# Replace the logic with something like
# If we have a multiline string
if token.string[0:3] in self.MULTILINE_QUOTE_STRS:
    # If we are ignoring multiline strings, then skip it
    if not self.multiline_quotes:
        continue
    # Otherwise, validate it
    elif token.string.startswith(self.multiline_quotes['good']):
      continue
```